db/seeds.rb

```diff
@@ -148,15 +148,17 @@ def dates_skipping_most_weekends(first_date: Date.current - 60.days,
 # create Attendance records, some random amount of part and full days.
 def make_attendance(first_date: Date.current - 10,
                     last_date: Date.current,
-                    earliest_checkin_hour: 7)
+                    earliest_checkin_hour: 7,
+                    child: Child.first)
 
   days_attended = dates_skipping_most_weekends(first_date: first_date, last_date: last_date)
   days_attended.each do |day_attended|
     random_checkin_time = (earliest_checkin_hour * 60) + Random.rand(60 * RAND_CHECKIN_HRS_RANGE).minutes
     random_checkout_time = random_checkin_time + Random.rand(60 * RAND_CHECKOUT_HRS_RANGE).minutes
 
-    Attendance.find_or_create_by!(check_in: day_attended + random_checkin_time,
-                                  check_out: day_attended + random_checkout_time)
+    # Attendances are a type of BillableOccurrence so we should always create them this way
+    BillableOccurrence.find_or_create_by!(child_approval: child.child_approvals[0], billable: Attendance.create!(check_in: day_attended + random_checkin_time,
```

Comment:
This is the model of how it "feels right" to create an attendance.  It's a Billable Occurrence that belongs to a child_approval (formerly: child_case_cycle), and it's of type Attendance.